### ISSUE TITLE
Add public submission guidelines for the pattern directory

### ISSUE THREAD

**Issue by angelasjin** [WordPress/pattern-directory#394]
Can we add some submission guidelines for the block pattern directory? I think that using what @shaunandrews [suggested as a starting point](https://www.figma.com/file/bz4qs4jZCkqvQzPH2fmX6C/Pattern-Directory?node-id=2257%3A0) would be great.

**Comment by StevenDufresne** [comment]
As an MVP could we add a simple step in the publish window as opposed to a full page?

Something like:
<img src="https://d.pr/i/vk3Yi6.png" width="400px">

**Comment by ryelle** [comment]
I think that's a good additional step, but we'll also want a standalone page for people who want to know what the rules are before creating anything.

**Comment by ryelle** [comment]
I added the content from the figma to https://wordpress.org/patterns/about/ (It definitely needs CSS help, this is just to get it in place). This is mostly about how to create a pattern, which is good, but isn't stuff like "don't spam"/"don't use trademarks". The [plugin directory info page](https://wordpress.org/plugins/developers/) links off to [more detailed guidelines](https://developer.wordpress.org/plugins/wordpress-org/detailed-plugin-guidelines/), that could work here too. Will the Pattern Directory have a handbook?

**Comment by StevenDufresne** [comment]
I would hope that the pattern directory wouldn't need a handbook. In-line documentation would be a better experience.

**Comment by ryelle** [comment]
I should have put that as a separate question- I don't mean a separate handbook for just using the pattern creator, I mean like the ["The Plugin Directory" section here](https://developer.wordpress.org/plugins/wordpress-org/) — detailed guidelines, troubleshooting, FAQs, etc. Trying to see where a potential "Detailed Guidelines" page would live.

Inline documentation will work for basic guidance, a pattern author shouldn't _need_ to read a handbook before starting. But I'm sure we'll get questions that can't be answered succinctly with inline docs.

**Comment by StevenDufresne** [comment]
> Inline documentation will work for basic guidance, a pattern author shouldn't need to read a handbook before starting. But I'm sure we'll get questions that can't be answered succinctly with inline docs.

Ok. Do we need a new standalone page at all then? Could we not just leverage the handbook? I feel like I've seen a `Do's` and `Dont's` section in the handbook, although I can't seem to find it. Maybe keep the docs close to the Block Editor 
documentation?

**Comment by ryelle** [comment]
We don't have a handbook yet, if we want to create one we'll need to decide where it goes/what goes in it. I still think having [the About page ](https://wordpress.org/patterns/about/) is useful, and until we have a handbook, we can post detailed guidelines at `https://wordpress.org/patterns/about/guidelines`.

This issue is for creating those "detailed guidelines" & the content on the About page. If we want to talk about a handbook, let's make a new issue for that.

**Comment by tellyworth** [comment]
I think for launch we should just add a few minimal catch-all guidelines to https://wordpress.org/patterns/about/, essentially saying:

* You must be the copyright owner of everything you submit
* Don't be a jerk
* We reserve the right to remove patterns for any reason

The [plugin directory guidelines](https://developer.wordpress.org/plugins/wordpress-org/detailed-plugin-guidelines/) have some text we could borrow (eg the morally offensive bit). Probably just a sentence for each would be enough. We can involve the community in developing detailed guidelines over time.

**Comment by angelasjin** [comment]
+1 to @tellyworth's suggestion. Let's use this as copy?

> People will be able to use submitted patterns without cost or attribution. To submit a pattern, you must have the copyright or other legal right for anything you submit. Remember, WordPress is family friendly! Patterns that include anything illegal, dishonest, or morally offensive will be removed. The Block Pattern Directory is moderated by volunteers, and we reserve the right to reject or remove submissions for reasons not explicitly listed. 

Not sure where best to put this on the About Patterns page.  Perhaps under "What is a WordPress pattern?"

**Comment by ryelle** [comment]
I dropped that onto [the About page](https://wordpress.org/patterns/about/). 

Also, I have a branch locally to tweak the styles on this page a bit, but it needs #428 to be merged first.

**Comment by ryelle** [comment]
@angelasjin @tellyworth Should we leave this issue open for any continued About and/or Guideline discussion, or do you think this is good for launch? Follow up discussion could happen in other issues, if needed.

**Comment by angelasjin** [comment]
I think this is good for launch!
